### ISSUE TITLE
lodash@<2.0.0 is no longer maintained

### ISSUE THREAD

**Issue by trdarr** [NickCarneiro/curlconverter#9]
```
$ npm install
npm WARN deprecated lodash@1.0.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
```

**Comment by trdarr** [comment]
After adding `grunt` as an explicit `devDependency` (see #11):

```
$ npm install
npm WARN deprecated lodash@0.9.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
npm WARN deprecated lodash@1.0.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
```

`grunt@0.4.5` depends on `lodash@0.9.2`, but that’s fixed? (gruntjs/grunt#1419).

`lodash@1.0.2` is a transitive dependency from `grunt-contrib-watch@0.6.1`:

```
+-- grunt-contrib-watch@0.6.1
| +-- async@0.2.10
| +-- gaze@0.5.2
| | `-- globule@0.1.0
| |   +-- glob@3.1.21
| |   | `-- inherits@1.0.2
| |   +-- lodash@1.0.2
| |   `-- minimatch@0.2.14
| +-- lodash@2.4.2
| `-- tiny-lr-fork@0.0.5
|   +-- debug@0.7.4
|   +-- faye-websocket@0.4.4
|   +-- noptify@0.0.3
|   | `-- nopt@2.0.0
|   `-- qs@0.5.6
```

Closing because these are transitive dependencies from up-to-date packages. :(

**Comment by NickCarneiro** [comment]
I'm not married to grunt. If there's some new build system you want to try out I won't stand in your way.
